lms/djangoapps/instructor_task/migrations/0005_alter_instructortaskschedule_task.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 3.2.13 on 2022-04-26 13:32
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('instructor_task', '0004_historicalinstructortaskschedule_instructortaskschedule'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='instructortaskschedule',
+            name='task',
+            field=models.OneToOneField(on_delete=django.db.models.deletion.DO_NOTHING, to='instructor_task.instructortask'),
```

Comment:
Changing `on_delete` functionality from CASCADE --> DO_NOTHING. 

My thoughts here are that we don't actually want to delete the task information, just the schedule. We can keep the remaining task info as an audit trail of actions taken by instructor/course staff.

Instead, we will update the status of the task to `REVOKED` when the schedule is deleted. This logic appears later in the PR.